lib/traject/numismatic_csv_config.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Numismatics CSV Mapping Configuration
+
+require_relative 'csv_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/dlme'
+require_relative 'macros/csv'
+extend Macros::DLME
+extend Macros::Csv
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'CsvReader'
+  provide 'allow_empty_fields', true
```

Comment:
Can you remove the `allow_empty_fields` setting? I don't think we want/need it set.